playwright.config.ts

```diff
@@ -0,0 +1,119 @@
+import { devices, PlaywrightTestConfig } from "@playwright/test";
+
+const config: PlaywrightTestConfig = {
+  testDir: "./e2e/tests/features",
+  outputDir: "./e2e/test-results",
+  testMatch: /features/,
+  testIgnore: "**/src/**",
+  reporter: [
+    ["list", { printSteps: true }],
+    ["html", { outputFolder: "./e2e/test-report" }],
+  ],
+  globalSetup: "./e2e/hooks/GlobalSetup.ts",
+  timeout: 60000,
+  workers: 6, // to enforce parallel workers in Actions Workflow
+  retries: 2,
+  projects: [
+    {
+      name: "BPoR-VS",
+      testIgnore: "**/accessibilityTests/**",
```

Comment:
From what I can see there are no accessibility tests, so you can remove this line if you want